packages/cli/src/cli.ts

```diff
@@ -35,117 +49,47 @@ export async function main(originalArgs: readonly string[]): Promise<void> {
-        "Secp256k1",
-        "Sha256",
-        "sha256",
-        "Sha512",
-        "sha512",
-        "Slip10",
-        "Slip10Curve",
-        "Slip10RawIndex",
-      ],
-    ],
-    [
-      "@cosmjs/encoding",
-      ["fromAscii", "fromBase64", "fromHex", "fromUtf8", "toAscii", "toBase64", "toHex", "toUtf8", "Bech32"],
-    ],
-    ["@cosmjs/faucet-client", ["FaucetClient"]],
-    [
-      "@cosmjs/launchpad",
-      [
-        "coin",
-        "coins",
-        "decodeAminoPubkey",
-        "decodeBech32Pubkey",
-        "encodeAminoPubkey",
-        "encodeBech32Pubkey",
-        "encodeSecp256k1Pubkey",
-        "encodeSecp256k1Signature",
-        "logs",
-        "makeCosmoshubPath",
-        "makeSignDoc",
-        "makeStdTx",
-        "IndexedTx",
-        "BroadcastTxResult",
-        "Coin",
-        "CosmosClient",
-        "GasLimits",
-        "GasPrice",
-        "Msg",
-        "MsgDelegate",
-        "MsgSend",
-        "LcdClient",
-        "OfflineSigner",
-        "PubKey",
-        "pubkeyToAddress",
-        "Secp256k1HdWallet",
-        "Secp256k1Wallet",
-        "SigningCosmosClient",
-        "StdFee",
-        "StdSignDoc",
-        "StdTx",
-      ],
-    ],
-    ["@cosmjs/math", ["Decimal", "Int53", "Uint32", "Uint53", "Uint64"]],
-    ["@cosmjs/utils", ["assert", "arrayContentEquals", "sleep"]],
-  ]);
-
   console.info(colors.green("Initializing session for you. Have fun!"));
-  console.info(colors.yellow("Available imports:"));
-  console.info(colors.yellow("  * axios"));
-  console.info(colors.yellow("  * fs"));
-  for (const [moduleName, symbols] of imports.entries()) {
-    console.info(colors.yellow(`  * from ${moduleName}: ${symbols.join(", ")}`));
-  }
-
-  let init = `
-    import axios from "axios";
-    import * as fs from "fs";
-  `;
-  for (const [moduleName, symbols] of imports.entries()) {
-    init += `import { ${symbols.join(", ")} } from "${moduleName}";\n`;
-  }
+  const visiblePackages = (await installedPackages()).filter(
+    (name) => name.startsWith("@cosmjs/") || name === "axios",
+  );
+  console.info(colors.yellow("The following packages have been installed and can be imported:"));
+  console.info(colors.yellow(visiblePackages.join(", ")));
 
+  let init = "";
   if (args.selftest) {
     // execute some trival stuff and exit
     init += `
+      import axios from "axios";
+      import * as fs from "fs";
+
+      import {
+        fromAscii,
+        fromBase64,
+        fromHex,
+        fromUtf8,
+        toAscii,
+        toBase64,
+        toHex,
+        toUtf8,
+        Bech32,
+      } from "@cosmjs/encoding";
+      import { sha512, Bip39, Random } from "@cosmjs/crypto";
+      import {
+        coins,
+        encodeAminoPubkey,
+        encodeBech32Pubkey,
+        decodeBech32Pubkey,
+        decodeAminoPubkey,
+        makeCosmoshubPath,
+        makeSignDoc,
+        Secp256k1HdWallet,
+        Secp256k1Wallet,
+        StdFee,
+      } from "@cosmjs/launchpad";
```

Comment:
Maybe some stargate imports here (or in the example scripts)

Reply:
I add an example script which talks to the chain. This can then be executed conditionally.

Reply:
Stargate example added in the last commit. Thanks